test/views/ObligedEntityPdfYesNoViewSpec.scala

```diff
@@ -68,11 +62,29 @@ class ObligedEntityPdfYesNoViewSpec extends YesNoViewBehaviours {
       "details.bullet13",
       "details.bullet14",
       "details.bullet15",
-      "subheading",
+      "subheading1",
+      "p3",
+      "bullet3",
+      "bullet4",
+      "bullet5",
+      "bullet6",
+      "p4",
+      "bullet7",
+      "bullet8",
+      "bullet9",
+      "bullet10",
       "p5",
+      "bullet11",
+      "bullet12",
+      "bullet13",
+      "subheading2",
       "p6",
-      "bullet7",
-      "bullet8"
+      "p7",
+      "bullet14",
+      "bullet15",
+      "p8",
+      "question"
+
```

Comment:
I think you can also add `yes` and `no` to this list. Also may as well name the parameters given we are parsing in a pretty messy list:
```scala annotate
    behave like normalPageTitleWithCaption(
      view = applyView(form),
      messageKeyPrefix = messageKeyPrefix,
      captionKey = "obligedEntityPdfYesNo",
      captionParam = "",
      ...

```